### ISSUE TITLE
hg log produces "Unrecognized ANSI SGR code <95>" with TERM=screen-256color

### ISSUE THREAD

**Issue by rameshdharan** [walles/moar#24]
```
$ hg log
Please post the following report at <https://github.com/walles/moar/issues>,
or e-mail it to johan.walles@gmail.com.

Version: 0.9.21
LANG   : en_US.UTF-8
TERM   : screen-256color

GOOS    : linux
GOARCH  : amd64
Compiler: gc
NumCPU  : 32

time="2020-04-22T00:37:36Z" level=warning msg="Unrecognized ANSI SGR code <95>"
time="2020-04-22T00:37:37Z" level=warning msg="Unrecognized ANSI SGR code <95>"
time="2020-04-22T00:37:37Z" level=warning msg="Unrecognized ANSI SGR code <95>"
time="2020-04-22T00:37:37Z" level=warning msg="Unrecognized ANSI SGR code <95>"
```

**Comment by walles** [comment]
Do you have an example output file?

I can handle this either way, but it would be nice to add an example file to my test suite so this won't accidentally break in the future.

**Comment by walles** [comment]
Just released 0.9.22 with this fix in it:
<https://github.com/walles/moar/releases/tag/0.9.22>

If you do have an example `hg log` output that you could share that would still be nice to have!
